### DISCUSSION TITLE
Contratação PJ - Negociação de Remuneração

### DISCUSSION THREAD

**Discussion by ericvvf** [frontendbr/forum#1374] [Discussão]
Boa noite, amigos!

Quando vocês vão negociar uma remuneração para uma posição de prestador de serviço PJ, costumam fazer aquela _conversão_, somando o que desejam ganhar como CLT + impostos (aproximadamente) que o contratante não paga neste modelo de contratação?

**Comment by GustavoGottardi** [comment]
Bom dia @ericvvf,
Eu trabalho como PJ e a dica que dou, caso seja sua primeira vez trabalhando como PJ é tenha um salário que deseja ganhar em mente, apartir desse salário adicione:

- INSS (Tem alíquotas de 8%, 9%, 11% e valor fixo de alíquota tudo depende do seu salário bruto), eu sempre coloco a alíquota máxima permitida que é de R$ 642,34
- FGTS (8% sobre o salário líquido)
- Férias (Pegue o salário divida por 12 meses)
- 13º Salário (Pegue o salário divida por 12 meses)
- Gastos com alimentação, normalmente cálculo 30,00 por dia (30 * 21 dias úteis = R$ 630,00)
- Gasto com transporte (Metrô, Trem, Ônibus, Gasolina)
- Gasto com Plano de Saúde (Valor depende do plano então pesquise antes)
- 6% da Nota Fiscal caso sua empresa seja enquadrada no Simples Nacional com o CNAE que inicia em 6% o meu por exemplo é o CNAE: 6209-1/00.

Além do que citei acima de gastos, eu sempre adiciono uns 30% a mais sobre o total pela instabilidade que é esse modelo, pois você está empregado em um dia e pode ser desligado no outro dia sem problema algum e sem ter garantias á receber.

Normalmente é isso o que faço e não tenho me arrependido até hoje, espero ter ajudado!

Abraços

**Comment by ericvvf** [comment]
Obrigado pela resposta, @GustavoGottardi! Hoje eu trabalho como PJ (100% remoto), então tem alguns gastos ai que não pus na minha negociação. Mas minha real dúvida, é se a galera meio que aumenta o valor que acha que vale (digamos, o cara ganha 4k CLT, vai la e cobra 7K com a ideia de que o empregador não vai ter alguns gastos com impostos).

**Comment by GustavoGottardi** [comment]
Entendi @ericvvf, realmente se você trabalha remoto gastos como alimentação e transporte não fazem sentido para você, mas o restante imagino que sim. E sim imagino que a galera cobre mais sendo PJ realmente para compensar "benefícios" que o CLT dá por padrão e que não existem no PJ, porém no PJ você tem o dinheiro na mão e pode gastar como quiser. :)

**Comment by cauequeiroz** [comment]
Cara, pelo que eu tenho visto com os meus colegas vai depender de qual modelo ele esta acostumado a trabalhar.

Se o cara sempre trabalhou como PJ, então o valor vai ser simplesmente baseado na progressão de carreira dele (começou ganhando 3k, depois 5k etc etc, ai para a o proxima proposta ele quer ganhar 8k).

Se o cara trabalhou como CLT, ai sim ele faz essa "conversão". No ultimo trabalho ele ganhava 5k CLT, então ele precisa ganhar 7k como PJ para ficar a mesma coisa, mas como ele quer um aumento então ele vai pedir 8.5k, por exemplo.

Se a pessoa nunca trabalhou como CLT e só é oferecido a opção de PJ, na minha opinião não faz nem sentido usar como base o suposto valor que ganharia de fosse CLT e etc...não tem referencial, é mais simples só pensar quanto vale o seu trabalho e pedir o salario que achar melhor.

**Comment by EduardoRFS** [comment]
Primeiro, só para avisar, ninguém vai te contratar por mais do que gastaria na CLT, pela insegurança juridica.

Segundo, não contabilize "direitos" que você nunca vai usar. Desculpa, se vc tem menos de 40 anos, você nunca vai se aposentar, INSS é apenas dinheiro queimado.

Terceiro, algumas paradas como FGTS, 5% na sua mão valem mais do que 8% na mão do governo.

Não esqueça que você pode pedir para copiar algumas paradas da CLT como férias, é muito bom se possível colocar uma garantia no contrato de 15-30 dias de férias anuais mesmo que não remunerados. No meu contrato atual tem uma clausula que preciso receber 15 dias de aviso prévio remunerado para poder ser demitido, eu também preciso dar um aviso de 15 dias antes, mas isso é vantajoso tanto para mim quanto para meu empregador.

Lembrando contrato como PJ é uma negociação, se você conseguir diminuir o custo para empresa e aumentar seu salario, você ta ganhando "ah mas a empresa ta gastando 1000 reais a menos", é e você ta ganhando 1500 a mais Lembre-se sempre do custo de oportunidade.

**Reply by guhmerces** [reply]
Sábio!

**Comment by kazzkiq** [comment]
@EduardoRFS Ainda não li essa nova proposta de reforma da previdência, mas concordo que a não ser que eles tenham feito algum milagre, também acho bem complicado contar com a aposentadoria "pelo governo".

Mas pra mim isso não quer dizer que você não deva cobrar a % do INSS, já que você pode usar esse valor em uma previdência privada, por exemplo. Se você for mais imediatista dá inclusive pra somar o valor do FGTS + INSS pra criar uma "poupança" a curto prazo, que pode servir de auxílio desemprego no futuro, ou pra ajudar a comprar um carro, casa, viagem, etc. Por isso acho válido colocar esses custos na conta sempre que possível, mesmo que você não vá contribuir diretamente pra previdência.

**Reply by guhmerces** [reply]
Sábio também !

**Comment by EduardoRFS** [comment]
O meu ponto é, se você quiser tudo que a CLT te dá, você não vai ser contratado como PJ, a não ser que você seja um funcionário muito requisitado, dai você pode negociar 300 coisas..

Sobre a reforma, ela funciona, mas ela da no máximo uns 10 anos a mais, no estado atual nossa previdência deve explodir em no máximo 20 anos, ou seja, mesmo quem tem 40 hj, na hora de se aposentar a previdência estaria explodindo já.

**Comment by ericvvf** [comment]
Muito obrigado pelas respostas, galera! Como falei acima, hoje eu trabalho como PJ, e estou tendo a oportunidade de participar, além do desenvolvimento, do processo de contratação e análise de candidatos. Daí surgiu um feeling bem pessoal de que algumas pessoas setam essa "conversão" quando fazem a pedida salarial, pois há as vezes uma desconexão entre o valor pedido e o que realmente o candidato mostra de valor técnico. Mas é só um achimos baseado num padrão que percebi (claro, uma amostra pequena, por isso quis gerar esta discussão aqui).

**Comment by NicholasKuchiniski** [comment]
Infelizmente para algumas empresas o PJ só serve para contratar um funcionário mais barato. Fantástico o comentário do @GustavoGottardi, com certeza levarei esses pontos em consideração no próximo contrato que eu fizer.
